cygnus-common/spec/SPECS/cygnus-common.spec

```diff
@@ -169,6 +169,9 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Fri Dec 02 2016 Francisco Romero <francisco.romerobueno@telefonica.com> 1.6.0
+- [cygnus][hardening] Add IPv6 support to API and GUI (#1261)
```

Comment:
I have just seen https://github.com/telefonicaid/fiware-cygnus/pull/1337/files#diff-09eccda12e4412bcaf14f1e078a741a9R100 below... Thus, I understand this is correct.